### DISCUSSION TITLE
Feature which I miss a lot -- Multi user support

### DISCUSSION THREAD

**Discussion by amgschnappi** [photoprism/photoprism#4333] [Polls]
I would like to have a single docker instance for my complete family - each having their own login.
This is so much missing for my use case.

**Comment by graciousgrey** [comment]
Users with different roles that work on one library can be created already. A more convenient way to manage multiple libraries is planned for the future:

- https://docs.photoprism.app/user-guide/users/libraries/

**Reply by temperatio** [reply]
Any ETA? 
I am satisfied with Photoprism overall, I even have the Photoprism+ version, but now I need a system that can be used by multiple users. I find it inconvenient to have photos from different users mixed together, and I consider having an instance per user to be inefficient in many aspects.

**Reply by graciousgrey** [reply]
Thank you  for your support and feedback!

We understand that many users are waiting for the Library Manager.

Issue 98 is a huge topic that includes many individual sub-features. Some of them are very time-consuming to develop, especially since security is very important when it comes to authentication and authorization.

Sub-features such as

- the own account page
- the user management for creating, editing and deleting users
- the user roles with restricted rights

were finalized last year.

The sub-features 2FA, app passwords and Open ID Connect support were released this year.

In the next step, we will implement the Library Manager and the feature to share albums/photos with guest users or users of other libraries via the Web UI. We will start development in Q4 and we expect to release these features in the first half of 2025.
